CHANGELOG.md

```diff
@@ -26,6 +26,8 @@ This preview version removes Crashes support for UWP. The App Center backend is
 
 ### App Center Crashes
 
+* **[Fix]** Fix an issue where stack traces for exceptions passed to `Crashes.TrackError` were incomplete.
```

Comment:
This line might need to be updated now (moved to different version's section)

Reply:
Yeah, please move it to the current version